src/connector/C#/README.md

```diff
@@ -0,0 +1,435 @@
+## CSharp Connector
+
+* This C# connector supports: Linux 64/Windows x64/Windows x86.
+* This C# connector can be downloaded and included as a normal package form [Nuget.org](https://www.nuget.org/packages/TDengine.Connector/).
+
+### Installation preparations
+
+* Install TDengine client, please refers to the[steps of installing connector driver](https://www.taosdata.com/en/documentation/connector#driver).
+* .NET interface file TDengineDriver.cs and reference samples both 
+  located under Windows client's installation path:install_directory/examples/C#.
+* Install [.NET SDK](https://dotnet.microsoft.com/download)
+
+### Installation verification
+
+Run {client_installation_directory}/examples/C#/C#Checker/C#Checker.cs
+Need to install .Net SDK first.
```

Comment:
.NET SDK is instructed in Installation preparations section, so this line is not necessary.